reference/release-notes/1.23.0.md

```diff
@@ -0,0 +1,93 @@
+---
+orphan: true
+---
+# 1.23.0
+
+The Anbox Cloud team is pleased to announce the release of Anbox Cloud 1.23.0.
+
+Please see [component versions](https://anbox-cloud.io/docs/reference/component-versions) for a list of updated components.
+
+## New features & improvements
+
+### Anbox Cloud Appliance
+The Anbox Cloud Appliance is overhauled to deliver a much simpler and streamlined version. This new version of the appliance is made available using the [snap epoch feature](https://snapcraft.io/docs/snap-epochs). Existing installations of the appliance are not affected but the old snap with `epoch=0` is {ref}`deprecated <sec-appliance-deprecation>`. For all new installations of the appliance, use the `epoch=1` snap. See the [appliance installation tutorial](https://documentation.ubuntu.com/anbox-cloud/en/latest/tutorial/installing-appliance/) for instructions.
+
+The `anbox-cloud-appliance` status command now displays the initialisation status of the appliance and its version. <!--AC-2513-->
+
+### Seamless ADB connections
+
+Anbox Cloud now offers a way to connect to remote Android instances securely using the Android Debug Bridge (ADB). You can set up an ADB connection using the dashboard or the CLI. See {ref}`howto-access-instance` for more information.
+
+### Full support for AAOS
+
+Android Automotive OS based images are fully supported starting from this release and no longer considered an experimental feature.<!--AC-2660-->
+
+### Anbox Cloud dashboard
+
+The Anbox Cloud dashboard has the following improvements:
+
+  * The *Instances* and the *Sessions* pages and their functionalities are merged to provide an enhanced user experience while streaming.
+
+  * The initial VHAL values upon joining a stream are now captured as a default. You can revert to these values using the *Reset* button, when needed.
+
+  * You can set up and authorise ADB connections for an instance from the dashboard.
```

Comment:
@morphis should there an item here for multi-cluster support here or no ?

Reply:
As per SU discussion, adding the known issue